parm/post_avblflds.xml

```diff
@@ -1510,10 +1510,8 @@
        <post_avblfldidx>171</post_avblfldidx>
        <shortname>MSTAV_ON_DEPTH_BEL_LAND_SFC</shortname>
```

Comment:
@EricJames-NOAA I wonder if you can add prefix "RUC" in the shortname such as RUC_MSTAV_ON_DEPTH_BEL_LAND_SFC for RUC LSM? Right now in all UFS based applications, only RRFS outputs MSTAV. We might need a new ID for MSTVA output from other LSMs in the future.

Reply:
@EricJames-NOAA From UFS model developer perspective, do you think MSTVA outputted from UFS could be labeled as "0 m underground", or just for RUC LSM?

Reply:
@WenMeng-NOAA I think this may be just for RUC LSM.  My understanding is that RRFS may move to another LSM in the future.  So it might be best to not restrict UFS to this 0 m underground labeling.

Reply:
@EricJames-NOAA Thanks for clarifying. Would you change the shortname of UPP id 171 for RUC? We might add new definition of MSTVA for other LSMs in the future.